install_config/install/advanced_install.adoc

```diff
@@ -1349,9 +1346,44 @@ set for a region called `primary` and a zone called `east`:
 node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 
+Starting in {product-title} 3.9, masters are now marked as schedulable nodes by
+default. As a result, the default node selector (defined in the master
+configuration file's `projectConfig.defaultNodeSelector` field to determine
+which node that projects will use by default when placing pods, and previously
+left blank by default) is now set by default during cluster installations. It is
+set to `node-role.kubernetes.io/compute=true` unless overridden using the
+`osm_default_node_selector` Ansible variable.
+
+In addition, as long as `osm_default_node_selector` is left unset, any
+non-master, non-dedicated infrastructure nodes hosts defined in your inventory
+file (for example, the `node1.example.com` host shown above) will also have the
+`node-role.kubernetes.io/compute=true` label set automatically on them during
+installation. This ensures that the default node selector has available nodes to
+choose from when determining pod placement.
+
+[IMPORTANT]
+====
+If you accept the default node selector of
+`node-role.kubernetes.io/compute=true` during installation, ensure that you do
+not only have dedicated infrastructure nodes as the non-master nodes defined in
+your cluster. In that scenario, application pods would fail to deploy because no
+nodes with the `node-role.kubernetes.io/compute=true` label would be available
+to match the default node selector when scheduling pods for projects.
+====
+
+If `osm_default_node_selector` is set to a custom value during installation,
```

Comment:
Per QE's testing in https://bugzilla.redhat.com/show_bug.cgi?id=1548641#c18, whatever osm_default_node_selector is set or NOT, this automatic node labeling always occurs. Master nodes are labeled with node-role.kubernetes.io/master=true, non-master, non-dedicated infrastructure nodes hosts (nodes with a region=infra label) are labeled with node-role.kubernetes.io/compute=true.